### ISSUE TITLE
IndexError: index out of range

### ISSUE THREAD

**Issue by olafmarcos** [ehn-dcc-development/ehn-sign-verify-python-trivial#13]
Testing the utility i get the following traceback and error:

echo '{"A": 1234}' | python3 hc1_sign.py | python3 hc1_verify.py

Traceback (most recent call last):
  File "hc1_sign.py", line 149, in <module>
    out = b'HC1:' + b45encode(out).encode('ascii')
AttributeError: 'bytes' object has no attribute 'encode'
Traceback (most recent call last):
  File "hc1_verify.py", line 153, in <module>
    if (cin[0] == 0x78):
IndexError: index out of range

python3 -V
Python 3.7.3

**Comment by bisinnoco** [comment]
I got the same problem

**Comment by senzacionale** [comment]
any solution?

**Comment by zeroninjahack** [comment]
Same problem here. Is there any solution for this error? 

out = b'HC1:' + b45encode(out) -> doesn't work for me.

I have another problem with: 

Traceback (most recent call last):
File "/Downloads/ehn-sign-verify-python-trivial/hc1_sign.py", line 149, in <module>
out = b'HC1:' + b45encode(out).encode('ascii')
AttributeError: 'bytes' object has no attribute 'encode'

**Comment by senzacionale** [comment]
Remove encode

**Comment by zeroninjahack** [comment]
> Remove encode 

@senzacionale 
Thank you for answering but, where should I remove encode? I don't understand your answer.
